tests/uitools/qml_quick/functional_tests/FloorFilterControllerFuncTest/tst_floorfiltercontroller.qml

```diff
@@ -0,0 +1,189 @@
+/*******************************************************************************
+ *  Copyright 2012-2022 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+
+import QtQuick 2.0
+import QtTest 1.0
+
+import Esri.ArcGISRuntime 100.14
+import Esri.ArcGISRuntime.Toolkit 100.14
+import Esri.ArcGISRuntime.Toolkit.Controller 100.14
+
+TestCase {
+    id: floorFilterControllerFuncTest
+    name: "FloorFilterControllerUnitTest"
+
+    Credential {
+        id: viewerCredentialIndoors
+        // need to add CREDENTIAL_USERNAME and CREDENTIAL_PASSWORD to the Environment variables.
+        username: credential_username
+        password: credential_password
+    }
```

Comment:
harmful in the sense of useless?
I could stick with the publicly available single site, single facility test. I liked running with the internal webmaps because are more complicated than the public one.
I could modify the tests to run for both the public and the private in case the credential environment are set, otherwise only run for the public. In this way, internally we can still run those more complicated ones! How does it sound?